README.md

```diff
@@ -389,6 +390,34 @@ Executive Trusonafications require the user to scan an identity document to auth
 
 [^1]: You must provide at least one field that would allow Trusona to determine which user to authenticate. The identifier fields are `DeviceIdentifier`, `TruCode`, `UserIdentifier`, and `EmailAddress`.
 
+
+
+### Canceling A Trusonafication
+
+Any Trusonafication that is `IN_PROGRESS` can be cancelled.
```

Comment:
What does a "cancelled" trusonafication mean? 

I think it would be helpful to add a sentence about what the act of canceling does and/or why you would want to cancel a trusonafication.